lib/Logger/LoggerFeature.cpp

```diff
@@ -556,6 +556,21 @@ printed in the thread that triggered the log message. This is non-optimal for
 performance but can aid debugging. If set to `false`, log messages are handed
 off to an extra logging thread, which asynchronously writes the log messages.)");
 
+  options
+      ->addOption(
+          "--log.max-queued-entries",
+          "Upper limit of log entries that are queued in background thread.",
+          new UInt32Parameter(&_maxQueuedLogMessages),
+          arangodb::options::makeDefaultFlags(
+              arangodb::options::Flags::Uncommon))
+      .setIntroducedIn(31200)
```

Comment:
May be adjusted for 3.11.
```suggestion
      .setIntroducedIn(31105)
```